tools/recetox_aplcms/macros.xml

```diff
@@ -93,52 +105,54 @@
-                       help="Relative retention time tolerance to use for grouping features." />
+                <param name="mz_tol_relative" type="float" optional="true" label="Relative m/z tolerance"
+                       help="Relative m/z tolerance to use for grouping features.
+                       If not provided, it is calculated from the data using kernel density estimation." />
+                <param name="rt_tol_relative" type="float" optional="true" label="Relative rt tolerance"
+                       help="Relative retention time tolerance to use for grouping features.
+                       If not provided, it is calculated from the data using kernel density estimation." />
             </when>
             <when value="file">
                 <param label="Input tolerances values" name="input_tolerances" type="data" format="parquet"
                        help="Table containing tolerance values." />
             </when>
         </conditional>
-        <param name="mz_tol_absolute" type="float" label="mz_tol_absolute" value="1e-05"
-               help="Absolute m/z tolerance to use for grouping features." />
-        <param name="mz_max_diff" type="float" label="mz_max_diff" value="0.01"
-               help="Maximum difference between feature m/z values to belong to the same cluster." />
+        <param name="mz_tol_absolute" type="float" label="Minimal absolute m/z tolerance [ppm]" value="10"
+               help="During the clustering, an m/z tolerance is computed based on the data and the specified relative tolerance.
+               This parameter allows the specification of a minimal value of this tolerance." />
+        <param name="mz_max_diff" type="float" label="Maximal m/z difference" value="0.01"
+               help="Maximum allowed difference between feature m/z values to belong to the same cluster." />
     </xml>
 
     <xml name="recover_weaker_params">
-        <param name="mz_tol" type="float" value="1e-05" label="mz_tol"
-               help="The m/z tolerance level for the grouping of data points. This value is expressed as the
-               fraction of the m/z value. This value, multiplied by the m/z value, becomes the cutoff level.
-               The recommended value is the machine's nominal accuracy level. Divide the ppm value by 1e6.
-               For FTMS, 1e-5 is recommended." />
-        <param name="recover_mz_range" type="float" optional="true" label="recover_mz_range"
+        <param name="mz_tol" type="float" value="10" label="m/z tolerance [ppm]"
+               help="The m/z tolerance level for the grouping of data points. This value is expressed as the fraction of the m/z value.
+               This value, multiplied by the m/z value, becomes the cutoff level.
+               The recommended value is the machine's nominal accuracy level (e.g. for FTMS, it is 1e-5)." />
+        <param name="recover_mz_range" type="float" optional="true" label="Range for m/z recovery"

```

Comment:
What is the unit of this?

Reply:
yeah - it should be daltons? Or is it ppm?

Reply:
I suppose it should have the same unit as `mz_tol`, i.e. ppm?

Reply:
yes, it should be - if it isn't passed, it is initialized as 1.5 * mz_tol_relative